### ISSUE TITLE
Quarto VSCode - No images preview since 1.86 (only encoding)

### ISSUE THREAD

**Issue by andreatitolo** [quarto-dev/quarto#197]
As per title, I am not able to preview images anymore when hovering with the mouse over the image path. Instead, I get a popup with some sort of encoding. I haven't found any settings that enable/disable this. Image format doesn't really matter and neither a trailing '/' in the path.

Attached a screenshot below

![quarto-vscode-image-preview](https://github.com/quarto-dev/quarto/assets/61151275/5663275d-14ac-440e-9ed7-12d016555a7d)

**Comment by jjallaire** [comment]
Thanks for reporting this! We moved image hover handling into our LSP process and it appears as if this imposes some limits on image size. We've reverted to our previous handling here: https://github.com/quarto-dev/quarto/commit/473cce5f8917963c332500716d32f5704e26aad1

This will be available in the next release of the extension (likely within a week or two)

**Comment by andreatitolo** [comment]
Thank you! 😄 
Sorry just a clarification, from the commit I see there is a comment about making it work for more than just png images. I do extensive use of webP format (to limit repo size), will other format be supported too in the next release?

**Comment by jjallaire** [comment]
Yes, `webp` and anything you can put in an `<img>` tag will indeed be supported.
